parser/model/ddl.go

```diff
@@ -447,8 +447,11 @@ type BackfillMeta struct {
 	WarningsCount map[errors.ErrorID]int64         `json:"warnings_count"`
 	Location      *TimeZoneLocation                `json:"location"`
 	ReorgTp       ReorgType                        `json:"reorg_tp"`
-
-	*JobMeta `json:"job_meta"`
+	RowCount      int64                            `json:"row_count"`
+	StartKey      []byte                           `json:"start_key"`
+	EndKey        []byte                           `json:"end_key"`
+	CurrKey       []byte                           `json:"curr_key"`
```

Comment:
Change `[]byte` to `kv.Key`? Unify it with the other key types.

Reply:
It is for ```parser``` to import tidb's package. it will happen cycle import.